### PR TITLE
GDB-5372 Fixed flaky tests and made improvements to test suite

### PR DESCRIPTION
**Pull request by yordanalexandrov**
- Removed some duplicate code
 - fixed issues with dropdown and inputs

### CODE REVIEW

File: cypress/steps/mapping-steps.ts [hunk 1]
```diff
@@ -287,7 +290,7 @@ class MappingSteps {
 
   // Notifications
   static getNotification() {
-    return cy.get('.mat-simple-snackbar');
+    return cy.get('.mat-simple-snackbar', {timeout: 10000});
```
Comment (svilenvelikov) [inline cypress/steps/mapping-steps.ts:293]: I don't like this. Unconditional wait usually means you don't know what happens with the app. Can't we have a proper check here?

File: cypress/test/triple-ending.spec.ts [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
-import MappingSteps from "../steps/mapping-steps";
+import PrepareSteps from '../steps/prepare-steps';
+import MappingSteps from '../steps/mapping-steps';
 
 describe('End triple with the proper sign', () => {
 
   beforeEach(() => {
     PrepareSteps.prepareMoviesNamespacesAndColumns();
-    cy.route('GET', '/orefine/command/core/get-models/?project=123', 'fixture:add-siblings/mapping-model.json');
+    cy.route('GET', '/orefine/command/core/get-models/?project=123', 'fixture:add-siblings/mapping-model.json').as('loadProject');
```
Comment (svilenvelikov) [inline cypress/test/triple-ending.spec.ts:8]: BTW you don't need to define these aliases `.as('loadProject')` when they are not used

**Review by svilenvelikov — APPROVED** [review]

File: cypress/steps/mapping-steps.ts (hunk 1 above)
Comment (yordanalexandrov) [reply cypress/steps/mapping-steps.ts:293]: The app is waiting for the toaster message to disappear. When under load it takes more than 4 sec (default timeout) to hide it. This timeout is not a wait. This is the maximum time cypress will wait for the toaster.

File: cypress/test/triple-ending.spec.ts (hunk 2 above)
Comment (yordanalexandrov) [reply cypress/test/triple-ending.spec.ts:8]: Yes, but this along with '@loadColumns' are used by `PrepareSteps#visitPageAndWaitToLoad`.

**Pull request merged**